spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala

```diff
@@ -72,8 +72,13 @@ private[connector] class CassandraRunner(val configTemplate: String) extends Emb
       copyTextFileWithVariableSubstitution(input, output, properties)
     }
   }
+  private def makeJarPath() : String ={
+    var sysClassLoader : ClassLoader = ClassLoader.getSystemClassLoader();
```

Comment:
var? looks like val

this is not java, no semi-colon at the end of a statement ;)
```ClassLoader.getSystemClassLoader```